cumulusci/tasks/github/tests/util_github_api.py

```diff
@@ -466,3 +467,94 @@ def _get_expected_label(self, name=None, desc=None):
 
     def _get_expected_labels(self, labels):
         return [self._get_expected_label(name=label) for label in labels]
+
+    def _get_expected_gist(self, description, files, public=False):
+        """Gist creationg returns 201 on success"""
+        gh_id = self.create_id(20)
+
+        gist_files = {}
+        for f in files:
+            for filename in f.keys():
+                gist_files[filename] = {
```

Comment:
🤔: Why pass a list of dicts? If you pass a dict you can iterate over `files.items()` instead.